src/services/multiversx-communication/mx.api.service.ts

```diff
@@ -202,10 +199,21 @@ export class MXApiService {
         from = 0,
         size = 100,
     ): Promise<EsdtToken[]> {
-        return this.doGetGeneric<EsdtToken[]>(
+        const userTokens = await this.doGetGeneric<EsdtToken[]>(
             this.getTokensForUser.name,
             `accounts/${address}/tokens?from=${from}&size=${size}`,
         );
+
+        for (const token of userTokens) {
+            if (!checkEsdtToken(token) || token.decimals === 0) {
```

Comment:
is there a reason why token.decimals === 0 this in not into checkEsdtToken? Also the name doesn't say to much

Reply:
done